01-intro.md

```diff
@@ -31,40 +29,54 @@ We want to:
 
 To do all that, we'll have to learn a little bit about programming.
 
-<div class="objectives" markdown="1">
-#### Objectives
-* Get to know the MATLAB environment.
-* Learn about MATLAB arrays.
-* Read tabular data from a file into a program.
-* Assign values to variables.
-* Select individual values and subsections from data.
-* Perform operations on arrays of data.
-* Display simple graphs.
-</div>
-
-### MATLAB GUI
-
-We have a dozen datasets that need analysis, stored as `.csv` files - but MATLAB doesn't know about these files yet. The first thing we need to do is set MATLAB's [working directory](../../gloss.html#current-working-directory) to the directory containing the files. To do this, we go to the `Home` tab, click on `Set Path`, and then on `Add with Subfolders...`. We navigate to the directory containing our files and add it to the path to tell MATLAB where to look for our files.
-
-Before we can start programming, we need to know a little about the MATLAB interface. 
+> ## Learning Objectives
```

Comment:
"Learning Objectives" is the first thing at the lesson. And you need to add the class "objectives":

~~~
> ## Learning Objectives {.objectives}
~~~